src/server/lib/middleware/login.ts

```diff
@@ -16,8 +15,8 @@ export const loginMiddleware = async (
   res: Response,
   next: NextFunction,
 ) => {
-  const config = getConfiguration();
-  const LOGIN_REDIRECT_URL = config.signInPageUrl;
+  const LOGIN_REDIRECT_URL =
+    process.env.SIGN_IN_PAGE_URL || 'https://profile.theguardian.com/signin';
```

Comment:
why do we need this fallback?

Reply:
Because `process.env.THING` could be undefined. In reality it won't be but typescript can't know that

Reply:
getConfiguration does not return an optional (since it throws an exception), can we use that?

Reply:
Yeah, good call. I moved away from that because this particular env var gets passed in via two methods (which confused me for a while!) but it turns out this is correct so I can revert to the original function

Reply:
Seems you're fixing this. But yeah, in agreement with Peter here, any environment variables required by the server should go through `getConfiguration` first.

Reply:
That's now done